ktor-client/ktor-client-features/ktor-client-tracing/ktor-client-tracing-stetho/src/androidMain/kotlin/io/ktor/client/features/tracing/StethoTracer.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.features.tracing
+
+import android.content.*
+import com.facebook.stetho.*
+import com.facebook.stetho.inspector.network.*
+import io.ktor.client.engine.*
+import io.ktor.client.request.*
+import io.ktor.http.cio.websocket.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.jvm.javaio.*
+
+/**
+ * Shortcut that allows to easily wrap engine factory into Stetho tracer.
+ */
+fun <T : HttpClientEngineConfig> Context.Stetho(delegate: HttpClientEngineFactory<T>): HttpClientEngineFactory<T> =
```

Comment:
Consider adding the test with this method

Reply:
Added.